src/System.Net.Sockets/tests/PerformanceTests/Perf.Sockets.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Net.Sockets.Tests;
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.Xunit.Performance;
+
+namespace System.Net.Sockets.Performance.Tests
+{
+    public class Perf_Sockets
+    {
+        /// <summary>
+        /// This test is built on top of the existing perf infrastructure and thus has some oddities that 
+        /// may be removed after some cleanup. I did not want to affect the structure of the SocketCommon any more than 
+        /// I had to, so I built this test around the existing API.
+        /// </summary>
+        [Benchmark]
+        [InlineData(0, 256, 100, SocketImplementationType.Async, SocketImplementationType.Async)]
+        [InlineData(0, 256, 1, SocketImplementationType.Async, SocketImplementationType.Async)]
+        [PlatformSpecific(PlatformID.Windows)] // Only tested on Windows&Linux and Linux fails with missing dll "ws2_32.dll"
```

Comment:
/cc @pgavlin 
I think this test worked fine in *NIX before. This is a regression?